### PR TITLE
Splashpage

### LINKED ISSUE #8: Create root Splashpage with navigation to Login, Signup, and Demo

### PR DESCRIPTION
**Pull request by sjstark**
Created splashpage. Features include:
- Includes links to `/users/login`, `/users/demo`, and `/users/signup`.
- Updated logos (including favicon.ico) to version 2.0, added 'task-doer' image (placeholder from original website)
- Added media query for example tasks
- CSS formatting

Closes #8

### CODE REVIEW

**Comment by sjstark** [comment]
I've updated the splashpage with a footer and links to out githubs as wells as some other locations.

One call out is that this does have a link to remember the milk because I was just looking for things to fill out the bars. I know we were told not to have pages link to the real app, but I'm curious if this was for features or if it's alright to have a link as a "reference". I'll ask Bryce.

File: views/error.pug [hunk 1]
```diff
@@ -1,6 +1,7 @@
 extends layout
```
Comment (memg92) [inline views/error.pug:1]: Should this be layout.pug? I'm assuming the '.pug' is not required if it's working on your end

Comment (sjstark) [reply views/error.pug:1]: Pug will assume that an extension is `.pug` if a file extension is not present. We could add `.pug` for clarity if you'd like though!

**Review by memg92 — APPROVED** [review]
Looks good to me. Only one comment when extending pug files – noticed in the error.pug file that it is not extending a '.pug' file, so I'm assuming file extension is not required?

**Pull request merged**